pom.xml

```diff
@@ -392,6 +392,12 @@
                 <groupId>org.antlr</groupId>
                 <artifactId>stringtemplate</artifactId>
                 <version>4.0.2</version>
+                <exclusions>
+                    <exclusion>
+                        <artifactId>antlr-runtime</artifactId>
+                        <groupId>org.antlr</groupId>
+                    </exclusion>
+                </exclusions>
```

Comment:
I'm still unclear why this exclusion would be necessary, since we're managing the `antlr-runtime` dependency version on line 225.

Reply:
It is because of a circular dependency. You end up with two versions of the same library being pulled in. The dependency management seems to fall over on a circular dependency on a managed dependency. If you remove this exclusion block (and you are DropWizard, or you have the enforcer plugin enabled yourself, which I have done), the build fails. `dropwizard-jdbi3` does not build on anything newer than 3.2.1 of JDBI as a result of the issue this fixes.